### DISCUSSION TITLE
DataSourceApi - panel resize

### DISCUSSION THREAD

**Discussion by vinnitu** [grafana/grafana#51363] [Data Sources]
Why my datasource plugin doesn’t get updated query (with new value of maxDataPoints) when panel was resized? Maybe I need implement some callback of DataSourceApi for this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
